### PR TITLE
Setup tuned profiles in /etc/tuned

### PR DESCRIPTION
**Pull request by jmencak**
This PR is to instantiate the new tuned profile hierarchy outlined in https://github.com/openshift/openshift-ansible/pull/5054.

### CODE REVIEW

**Comment by jmencak** [comment]
@sdodson Could you please take a look?  Thanks.

**Comment by sdodson** [comment]
I think you forgot to commit the actual profiles.

File: roles/openshift_node/tasks/main.yml [hunk 1]
```diff
@@ -70,25 +70,15 @@
     - openshift_disable_swap | default(true) | bool
 # End Disable Swap Block
 
-# We have to add tuned-profiles in the same transaction otherwise we run into depsolving
-# problems because the rpms don't pin the version properly. This was fixed in 3.1 packaging.
 - name: Install Node package
   package:
-    name: "{{ openshift.common.service_type }}-node{{ openshift_pkg_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }},tuned-profiles-{{ openshift.common.service_type }}-node{{ openshift_pkg_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }}"
+    name: "{{ openshift.common.service_type }}-node{{ openshift_pkg_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }}"
```
Comment (sdodson) [inline roles/openshift_node/tasks/main.yml]: The tuned-profiles-%{name}-node package is still a requirement of the node package in origin. We should probably remove that at the same time. If this doesn't get coordinated will the behavior here still take effect or do we need to wait on that to happen?

Comment (jmencak) [inline roles/openshift_node/tasks/main.yml]: The actual profiles were included in a separate PR: https://github.com/openshift/openshift-ansible/pull/5054, PTAL.  I agree that the rpm dependency should get removed from the origin repo (origin.spec file), but this is something I wanted to do after the fact (PRs 5054 and 5089 are merged).  The correct profiles (new profile hierarchy) will be used even if you leave the old tuned rpm packages as a dependency.  Tested both on RHEL and RHEL Atomic Host 7.4.

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge]

**Comment by slowbo** [comment]
@sdodson I did switch to ansible 2.3.x after I've noticed that the issue was related to the version but wanted to let you guys know that this problem exists :-) And my cluster is running ever since ;-)

Kind regards,
Slowbo

**Pull request merged**